lib/torque/animator.js

```diff
@@ -118,9 +118,15 @@ var cancelAnimationFrame = global.cancelAnimationFrame
       this._t0 = t1;
       this._time += delta;
       if(this.step() >= this.options.steps) {
-        this._time = 0;
         if(!this.options.loop){
-          this.stop();
+          // set time to max time
+          this.time(this.options.animationDuration);
+          // go to final frame (callback to nothing)
+          requestAnimationFrame(function(){});
```

Comment:
but calling time should render the last frame (https://github.com/abenrob/torque/blob/loop-pause-max/lib/torque/animator.js#L66)